code/modules/mob/living/simple_animal/abnormality/zayin/pile_of_mail.dm

```diff
@@ -0,0 +1,303 @@
+			if(ABNORMALITY_WORK_ATTACHMENT)
+				MAIL = new /obj/item/mailpaper/attachment(get_turf(src),user)
+	MAIL.throw_at(user, 4, 1, src, spin = FALSE, gentle = TRUE, quickstart = FALSE)
+	return
+//players get a parcel / letters for completing works
+/mob/living/simple_animal/hostile/abnormality/mailpile/PostWorkEffect(mob/living/carbon/human/user, work_type, pe, work_time, canceled)
+	. = ..()
+	if(work_type == ABNORMALITY_WORK_REPRESSION)
+		DeliveryRepress(user,work_type,pe)
+	return
+
+/mob/living/simple_animal/hostile/abnormality/mailpile/NeutralEffect(mob/living/carbon/human/user, work_type, pe)
+	. = ..()
+	Delivery(user,work_type,pe)
+	return
+
+/mob/living/simple_animal/hostile/abnormality/mailpile/SuccessEffect(mob/living/carbon/human/user, work_type, pe)
+	. = ..()
+	Delivery(user,work_type,pe)
+	return
+
+//what the fuck is this
+/obj/effect/letters_flow
+	name = "Storm of letters"
+	desc = "The storm rages on."
+	icon = 'ModularTegustation/Teguicons/32x32.dmi'
+	icon_state = "letters"
+	layer = ABOVE_MOB_LAYER
+
+/obj/effect/temp_visual/maildecal
+	name = "A letter"
+	desc = "You have a bad feeling about this."
+	icon = 'icons/obj/bureaucracy.dmi'
+	icon_state = "mail"
+	layer = BELOW_MOB_LAYER
+	duration = 10 SECONDS
+
+//x2 workspeed buff pro 2022 free hack free robux
+/datum/status_effect/workspeed_buff
+	id = "workspeed_buff"
+	status_type = STATUS_EFFECT_UNIQUE
+	alert_type = /atom/movable/screen/alert/status_effect/workspeed_buff
+	duration = 6 SECONDS
+
+/datum/status_effect/workspeed_buff/on_apply()
+	. = ..()
+	var/mob/living/carbon/human/user = owner
+	user.physiology.work_speed_mod *= 1.5
+
+/datum/status_effect/workspeed_buff/on_remove()
+	. = ..()
+	var/mob/living/carbon/human/user = owner
+	user.physiology.work_speed_mod /= 1.5
+
+/atom/movable/screen/alert/status_effect/workspeed_buff
+	name = "A letter from the past"
+	desc = "Reading the letter made you want to try harder for your past friends."
+	icon = 'icons/obj/bureaucracy.dmi'
+	icon_state = "mail"
+
+/datum/status_effect/nofear_buff
+	id = "nofear_buff"
+	status_type = STATUS_EFFECT_UNIQUE
+	alert_type = /atom/movable/screen/alert/status_effect/nofear_buff
+	duration = 6 SECONDS
+
+/datum/status_effect/nofear_buff/on_apply()
+	. = ..()
+	var/mob/living/carbon/human/user = owner
+	ADD_TRAIT(user, TRAIT_WORKFEAR_IMMUNE, TRAIT_GENERIC)
+	ADD_TRAIT(user, TRAIT_COMBATFEAR_IMMUNE, TRAIT_GENERIC)
+
+/datum/status_effect/nofear_buff/on_remove()
+	. = ..()
+	var/mob/living/carbon/human/user = owner
+	REMOVE_TRAIT(user,TRAIT_WORKFEAR_IMMUNE,TRAIT_GENERIC)
+	REMOVE_TRAIT(user,TRAIT_COMBATFEAR_IMMUNE,TRAIT_GENERIC)
+
+/atom/movable/screen/alert/status_effect/nofear_buff
+	name = "A letter from the present"
+	desc = "Reading the letter made you fearless."
+	icon = 'icons/obj/bureaucracy.dmi'
+	icon_state = "mail"
+
+//yes this is ripped off of paper and mermaid gift but fuck you.
+/obj/item/parcelself
+	name = "Parcel from your alternate self"
```

Comment:
```suggestion
	name = "parcel from your alternate self"
```
It should be `That's a parcel from your alternate self.` or `You hit yourself with the parcel from your alternate self`, not `That's Parcel from your alternate self.`/`You hit yourself with Parcel from your alternate self.`